cmd/metal-api/internal/service/tenant-service.go

```diff
@@ -72,6 +73,17 @@ func (r *tenantResource) webService() *restful.WebService {
 		Returns(http.StatusOK, "OK", []v1.TenantResponse{}).
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
+	ws.Route(ws.POST("/{id}/history").
+		To(viewer(r.getTenantHistory)).
+		Operation("getTenantHistory").
+		Doc("get tenant with this id at the given timestamp").
+		Param(ws.PathParameter("id", "identifier of the tenant").DataType("string")).
+		Metadata(restfulspec.KeyOpenAPITags, tags).
```

Comment:
Maybe it makes sense to exclude this from auditing as this is just a read operation:

```suggestion
		Metadata(restfulspec.KeyOpenAPITags, tags).
		Metadata(auditing.Exclude, true).
```